src/GraphQLParser/NodeHelper.cs

```diff
@@ -0,0 +1,416 @@
+using System.Runtime.CompilerServices;
+using GraphQLParser.AST;
+
+namespace GraphQLParser
+{
+    internal static class NodeHelper
+    {
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static GraphQLComment CreateGraphQLComment(IgnoreOptions options)
```

Comment:
First three methods differ from others. `GraphQLComment`, `GraphQLDocument` and `GraphQLDescription` have no comments. All other methods are completely the same. Perhaps when I will write tests, I will see that comments are not applicable to some other AST nodes.